src/components/form/HdDynamicForm.vue

```diff
@@ -52,12 +55,9 @@ export default {
     },
     lang: {
       type: String,
-      default: '',
+      default: undefined,
```

Comment:
What do you think about setting the de-facto default `de` here on the form level, as it is happening under the hood in the input components anyway?

Reply:
I actually set it to `undefined` for the purpose of always falling to the components' default. The components can be used individually (without the form component), so I wanted to avoid having default values on different levels, and make it clear that it's the components' defaults that we are using here.
To be honest, I also don't like having the `'de'` string literal everywhere 😅  we should have it defined in one place and maybe make it configurable on the library level as well. ([ticket created](https://homeday.atlassian.net/browse/FET-148))

Reply:
Ok, gotcha! That makes sense, and thanks for creating the ticket

Reply:
Thank you for the review! ❤️